agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/service/BaseDiseaseAnnotationService.java

```diff
@@ -107,14 +107,14 @@ protected boolean hasNoObsoletedOrInternalEntities(DiseaseAnnotation da) {
 			entitiesToBeValidated.addAll(da.getDiseaseGeneticModifiers());
 		}
 		AtomicBoolean hasNoObsoletedOrInternalEntities = new AtomicBoolean(true);
-		entitiesToBeValidated.forEach(auditedObject -> {
+		for(AuditedObject auditedObject: entitiesToBeValidated) {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'for' is not followed by whitespace.
<!-- __reviewdog__:Cg82Y2QxYTA4YTE0MGVlNjQSCmNoZWNrc3R5bGU= -->